core/src/main/scala/org/apache/spark/status/api/v1/api.scala

```diff
@@ -63,6 +63,7 @@ case class ApplicationAttemptInfo private[spark](
 
 class ExecutorStageSummary private[spark](
     val taskTime : Long,
+    val activeTasks: Int,
```

Comment:
You don't need to expose this in the public API to fix the bug, do you?

Reply:
Thank you @vanzin for the review.
Actually my objective is to get the last task of the particular executorId of the stage. If corresponding activeTasks == 0, then force update in the kvstore.

In stages, jobs, exec has "activeTasks" and using the parameter,  it force update on the last task.

https://github.com/apache/spark/blob/9a5fda60e532dc7203d21d5fbe385cd561906ccb/core/src/main/scala/org/apache/spark/status/AppStatusListener.scala#L631

https://github.com/apache/spark/blob/9a5fda60e532dc7203d21d5fbe385cd561906ccb/core/src/main/scala/org/apache/spark/status/AppStatusListener.scala#L563

Reply:
You didn't answer my question.

Reply:
Okay. I will try without exposing in the public API

Reply:
Hi @vanzin , I have modified based your comment. Kindly review